webapp/channels/src/components/channel_header/channel_header_title.tsx

```diff
@@ -107,8 +107,11 @@ const ChannelHeaderTitle = ({
                     aria-level={2}
                     id='channelHeaderTitle'
                     className='heading'
+                    aria-labelledby='channelHeaderTitle_bot'
```

Comment:
Why use `aria-labelledby` here when the inner span is already the child of this? I'd think that the readout for this would just be the the labels of `archivedIcon` (if it has one) and `channelTitle`